packages/appstore/src/constants/tour-steps-config.tsx

```diff
@@ -166,6 +166,118 @@ export const getWalletStepConfig = (): Step[] => [
         spotlightPadding: 8,
         styles: { spotlight: { borderRadius: '4.8rem' } },
     },
+    {
+        title: (
+            <React.Fragment>
+                <Text as='p' weight='bold' color='brand-red-coral'>
+                    {localize('CFDs trading accounts')}
+                </Text>
+                <div className='toggle-account-type__divider' />
+            </React.Fragment>
+        ),
+        content: (
+            // TODO: update this component if there's a cfd account
```

Comment:
@aizad-deriv this step and also the `Explore more wallets` need to be conditional, where did you cover those conditions?

Reply:
@mahdiyeh-deriv for the CFD part I am currently blocked by Thisyahlen's card hence why I added the temporary comment there. I completely missed the Add More Wallet part. I will add it in my next commit. 🙇🏼